src/main/java/com/karimbouchareb/chromatographysimulator/ui/SplashScreen.java

```diff
@@ -0,0 +1,206 @@
+package com.karimbouchareb.chromatographysimulator.ui;
+
+import com.karimbouchareb.chromatographysimulator.ChromatographySimulatorApp;
+import javafx.animation.FadeTransition;
+import javafx.animation.RotateTransition;
+import javafx.animation.TranslateTransition;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+import javafx.util.Duration;
+import org.kordamp.ikonli.javafx.FontIcon;
+import org.kordamp.ikonli.materialdesign.MaterialDesign;
+
+import static com.karimbouchareb.chromatographysimulator.ChromatographySimulatorApp.SCREEN_BOUNDS;
+import static com.karimbouchareb.chromatographysimulator.ChromatographySimulatorApp.makeImageView;
+
+public class SplashScreen {
+    // TOP-LEVEL FIELDS
+    public static StackPane splashScreenPane = new StackPane();
+    private static ImageView title;
+    private static Pane titlePane;
+    private static Pane baselineAnimationPane;
+    private static Pane travisHeadPane;
+    private static ImageView backgroundScatter;
+    private static Pane launchPane;
+    private static ImageView travisHead;
+
+    // TOP-LEVEL STATIC METHODS -- LOAD EACH UI COMPONENT OF THE SPLASH SCREEN
+    public static void splashScreenInit(){
+        splashScreenPane.setBackground(Background.fill(Color.BLACK));
+        backgroundInit();
+        travisHeadInit();
+        titleInit();
+        baselineAnimationInit();
+        launchButtonInit();
+        // Place background, travis, baselineAnimation, title, and launch button into scene
+        splashScreenPane.getChildren().addAll(backgroundScatter, travisHeadPane, baselineAnimationPane, titlePane, launchPane);
+    }
+
+    static void titleInit(){
```

Comment:
Let's rearrange these methods to match the order in which they are called, esp since they are sequential. Also, give brief Javadoc about what each method does.